### PR TITLE
Add ibm diffprivlib

### PR DESCRIPTION
**Pull request by oblute**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by oblute** [comment]
@rluria14 @ndmaxar agree to be maintainers?

File: recipes/diffprivlib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - diffprivlib
+    - diffprivlib.mechanisms
+    - diffprivlib.mechanisms.transforms
+    - diffprivlib.models
+    - diffprivlib.tools
+    - tests
+    - tests.mechanisms
+    - tests.mechanisms.transforms
+    - tests.models
+    - tests.tools
```
Comment (xhochy) [inline recipes/diffprivlib/meta.yaml:40]: Don't import check the tests
```suggestion
```

File: recipes/diffprivlib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8dba0a65ecd52c79b71d8f3837228386a154df7039fa149eb06bc48b707b9bfa
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```
Comment (xhochy) [inline recipes/diffprivlib/meta.yaml:15]: The parameters are no longer needed.
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

File: recipes/diffprivlib/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  host:
+    - pip
+    - python
+  run:
+    - joblib >=0.13.2
+    - numpy >=1.15.0
+    - python
+    - scikit-learn >=0.20.3
+    - scipy >=1.2.1
+    - setuptools >=39.0.1
```
Comment (scopatz) [inline recipes/diffprivlib/meta.yaml:27]: Is setuptools needed at runtime?

Comment (oblute) [reply recipes/diffprivlib/meta.yaml]: @scopatz nope, just removed it. thanks.

**Comment by oblute** [comment]
@scopatz @conda-forge/staged-recipes above changes made - ready for final review & merge.

**Pull request merged**